tools/export/vscode/launch.tmpl

```diff
@@ -40,7 +40,14 @@
                 "debugServerPath": "/usr/local/bin/pyocd-gdbserver"
             },
             "windows": {
-                "MIMode": "gdb"
+                "MIMode": "gdb",
+                "MIDebuggerPath": "${workspaceRoot}\\.vscode\\bin\\windows\\gcc\\arm-none-eabi-gdb.exe",
+                "debugServerPath": "${workspaceRoot}\\.vscode\\bin\\windows\\gdb_server.exe",
+                "setupCommands": [
+                    { "text": "-target-select remote localhost:3333", "description": "connect to target", "ignoreFailures": false },
+                    { "text": "-file-exec-and-symbols z:\\\\mbed-os-example-blinky\\\\BUILD\\\\${workspaceRootFolderName}.elf", "description": "load file", "ignoreFailures": false},
```

Comment:
Yep... GDB wants it's own escape for backslashes... And then escape in JavaScript, so that's 4. Very annoying.